static-site/content/blog/2025-07-18-rubella-resources.md

```diff
@@ -0,0 +1,96 @@
+---
+author: John SJ Anderson
+date: "2025-07-18"
+title: "New Resources for Rubella"
+sidebarTitle: "Rubella resources"
+---
+
+We are pleased to announce the availability of a Rubella phylogenetic
+analysis and dataset on Nextstrain.org, as well as a Rubella
+[Nextclade dataset][]. We provide phylogenetic trees based on both
+[whole genome] analysis, as well as analysis of [the E1 region] used in
+the [WHO genotype reference][]. These trees are automatically rebuilt
+with new Rubella sequences as they are deposited in NCBI. We welcome
+feedback from Rubella researchers about any aspect of either the
+Nextclade dataset or the phylogenetic trees on Nextstrain.org.
+
+## Phylogenetic analysis
+
+Rubella virus (_Rubivirus rubellae_) is a positive-strand RNA virus
+with a 9,762 base genome. It causes rubella, a formerly common
+respiratorily-transmitted illness, primarily infecting children. When
+rubella infects a pregnant person and spreads to the fetus, it causes
+congenital rubella syndrome (CRS), which can lead to a wide variety of
+congenital defects. The earlier in pregnancy infection happens, the
+more severe these defects tend to be. Rubella has been effectively
+eradicated in many parts of the world due to successful vaccination
+programs. Available vaccines provide lifelong immunity after a single
+dose, usually administered as part of the tri-valent "MMR" vaccine,
+which also protects against measles and mumps.
+
+The Rubella genome has two open reading frames (NSP and SP), the
+latter of which encodes three structural proteins, C (capsid), E1, and
+E2 (envelope proteins). E1 is the source of hemagglutin activity for
+the virus, and is the only region that is sequenced for many samples.
+It is also the basis of the [WHO genotype reference][], which we used
+for genotype definition in the Nextclade dataset.
```

Comment:
Consider adding an extra sentence here explaining the genotypes -- e.g. _n_ genotypes, the aim of which is to explain diversity?

Reply:
sentence added, thanks.